### ISSUE TITLE
updateLists DirectoryNotFoundException

### ISSUE THREAD

**Issue by BaHeK1994** [LTmadness/ValheimBetterServerConfig#24]
```
DirectoryNotFoundException: Could not find a part of the path "/home/server3270/game / bannedlist.txt".
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00164] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options, System.String msgPath, System.Boolean bFromProxy, System.Boolean useLongPath, System.Boolean checkHost) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions,string,bool,bool,bool)
  at System.IO.StreamWriter.CreateFile (System.String path, System.Boolean append, System.Boolean checkHost) [0x0001c] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize, System.Boolean checkHost) [0x00055] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.IO.StreamWriter..ctor (System.String path) [0x00008] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamWriter..ctor(string)
  at SyncedList.Save () [0x00000] in <5b89d6eaf15e446687a460b53a061654>:0 
  at SyncedList..ctor (System.String fileName, System.String defaultFileComment) [0x00051] in <5b89d6eaf15e446687a460b53a061654>:0 
  at ValheimBetterServerConfig.Console.Methods.UpdateFromFile (System.String[] args) [0x00066] in <aa929ca05043436db385d73e9edf541a>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <eae584ce26bc40229c1b1aa476bfa589>:0 
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at ValheimBetterServerConfig.Command.Run (System.String[] args) [0x00001] in <aa929ca05043436db385d73e9edf541a>:0 
  at ValheimBetterServerConfig.Runner.RunCommand (System.String text) [0x00078] in <aa929ca05043436db385d73e9edf541a>:0 
  at ValheimBetterServerConfig.ValheimBetterServerConfig.<Start>b__7_0 () [0x00045] in <aa929ca05043436db385d73e9edf541a>:0 
  at System.Threading.Tasks.Task.InnerInvoke () [0x0000f] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Threading.Tasks.Task.Execute () [0x00010] in <eae584ce26bc40229c1b1aa476bfa589>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at ValheimBetterServerConfig.ValheimBetterServerConfig+<Start>d__7.MoveNext () [0x00074] in <aa929ca05043436db385d73e9edf541a>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () [0x00002] in <51725b129ecd4967a9a213c1ae65b295>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
UnityEngine.WorkRequest:Invoke()
UnityEngine.UnitySynchronizationContext:Exec()
UnityEngine.UnitySynchronizationContext:ExecuteTasks()
```

Config parameter
```
## Server data save location
# Setting type: String
# Default value: 
Save location = /home/server3270/game
```

**Comment by LTmadness** [comment]
thank you, found the cause of the issue, will be included in next release( hopefully in next day or two)

**Comment by LTmadness** [comment]
fixed in the newest release v0.1.1
